packages/interfaces/src/message.ts

```diff
@@ -1,7 +1,7 @@
 import type { PubsubTopic } from "./misc.js";
 
 export interface SingleShardInfo {
-  clusterId: number;
+  clusterId?: number;
```

Comment:
instead of making this optional, i'd rather use `DEFAULT_CLUSTER_ID` that refers to `1` for TWN.

Reply:
let's do this a followup PR @adklempner

Reply:
@danisharora099 I made an update in my latest commit that makes all the fields required and instead uses `Partial<ShardingParams>` in all SDK create node functions, which are converted to regular `ShardingParams` before being used internally